Source/AlienRace/AlienRace/Graphic/ShaderDatabase.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Verse;
+using System.Collections.Generic;
+using UnityEngine;
+using System.Linq;
+using System.IO;
+using AlienRace
+namespace AvaliMod
+{
+    [StaticConstructorOnStartup]
+    public class AvaliShaderDatabase
+    {
+        static AvaliShaderDatabase()
+        {
+            string dir = AlienRace.AlienRaceMod.dir;
+
+            Log.Message(dir);
+            string path = dir + "/RimValiAssetBundles/shader";
+            AssetBundle bundle = RimValiUtility.shaderLoader(path);
+            Tricolor = (Shader)bundle.LoadAsset("assets/resources/materials/avalishader.shader");

```

Comment:
If tynan ever updates rimworld to a newer unity version, the shader may need to be updated as well.